roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -145,6 +145,19 @@ reserved_host_memory_mb=0
 {% elif inventory_hostname in groups['controller'] -%}
 reserved_host_memory_mb=4096
 {% endif %}
+
+{% if ceph.enabled and nova.backend_ceph -%}
+images_type = rbd
+images_rbd_pool = {{ ceph.vm_pool_name }}
+images_rbd_ceph_conf = {{ ceph.cinder.rbd_ceph_conf }}
+rbd_user = {{ ceph.cinder.rbd_user }}
+rbd_secret_uuid = {{ cinder_uuid_file.content | b64decode }}
```

Comment:
Upon referring: http://docs.ceph.com/docs/master/rbd/rbd-openstack/#configuring-nova-to-attach-ceph-rbd-block-device, the doc uses ```cinder_uuid``` in the ```nova.conf```. Hence we have not added a separate nova secret and used the same.